rollup.config.js

```diff
@@ -14,7 +14,7 @@ export default {
         {dest: 'dist/glamorous.es.js', format: 'es'},
         {dest: 'dist/glamorous.cjs.js', format: 'cjs'},
   ],
-  exports: 'default',
+  exports: 'named',
```

Comment:
Hmmm... you know what, I feel sorry about this, but I've changed my mind. It's probably better if we don't do that weird hacky stuff and just stick with what you had before. We'll push a breaking change. Sorry about that. Do you mind going back to what you had before?